### PR TITLE
🇧🇷 Brazil related commands

### PR DESCRIPTION
**Pull request by Doges**
- [x] CheckPlate (check the vehicle registration number)
- [x] CEP (search a cep using viacep)

### CODE REVIEW

**Comment by Doges** [comment]
spliting this into 4 prs bc the checkplate needs heavy testing

**Comment by Doges** [comment]
RFR

File: src/apis/ViaCEP.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+module.exports = class ViaCEP extends APIWrapper {
+  constructor () {
+    super({
+      name: 'viacep'
+    })
+  }
+
+  async searchCEP (cep) {
+          return this.request(cep)
```
Comment (metehus) [inline src/apis/ViaCEP.js:14]: identação ta como

Comment (metehus) [reply src/apis/ViaCEP.js:14]: I CLICKED APPROVE BY MISTAKE

Comment (Doges) [reply src/apis/ViaCEP.js:14]: da uma arrumada ai, to resolvendo os conflito

**Pull request merged**